apps/communications/contacts/test/marionette/export_sdcard_test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+var fs = require('fs');
+
+marionette('Export to SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
+        'device.storage.testing',
+        'device.storage.prompt.testing'];
+  prefs.forEach(pref => {
+    profile[pref] = true;
+  });
+  var client = marionette.client({
+    profile: profile,
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
```

Comment:
I'd put that under `Contacts.config` as this capabitilities is already defined in 8 other contacts test files.